google/cloud/network_services_v1/types/grpc_route.py

```diff
@@ -279,7 +279,6 @@ class Destination(proto.Message):
                   from the exact proportion defined here
                   depending on the precision an implementation
                   supports.
-
```

Comment:
This removal of blank line failed the "docs / docs" check.

```
sphinx.errors.SphinxWarning: /home/runner/work/python-network-services/python-network-services/google/cloud/network_services_v1/types/grpc_route.py:docstring of google.cloud.network_services_v1.types.grpc_route.GrpcRoute.Destination:26:Bullet list ends without a blank line; unexpected unindent.
```

Reply:
Googleapis-gen does have the empty line. https://github.com/googleapis/googleapis-gen/blob/master/google/cloud/networkservices/v1/networkservices-v1-py/google/cloud/network_services_v1/types/grpc_route.py

Why my environment removed the line...?

Reply:
I was referencing old revision of googleapis-gen https://github.com/googleapis/googleapis-gen/blob/104e9c5902f64afd75bdb6507a7702eaa7b43803/google/cloud/networkservices/v1/networkservices-v1-py/google/cloud/network_services_v1/types/grpc_route.py , which does not have this empty line.